components/scroll-to-top-button.tsx

```
@@ -0,0 +1,165 @@
+    
+    scrollToTop();
+  };
+
+  if (!isVisible) return null;
+
+  return (
+    <div className="fixed bottom-8 right-8 z-50">
+      {/* Progress ring for elegant variant */}
+      {variant === "elegant" && (
+        <div className="absolute inset-0 rounded-full">
+          <svg className="w-full h-full transform -rotate-90" viewBox="0 0 36 36">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)